ksp_plugin_adapter/localization_extensions.cs

```diff
@@ -54,27 +54,28 @@ internal static class L10N {
     return qualifiers == "" ? s : $"{s}^{qualifiers}";
   }
 
-  private static string NameWithoutArticle(this CelestialBody body) {
-    return StartsWithCapitalizedDefiniteArticle(body.displayName)
-        ? LingoonaQualify(
-              body.displayName.Split(new[]{' '}, 2)[1],
-              LingoonaQualifiers(body.displayName).Replace("d", ""))
-        : body.displayName;
-  }
-
-  public static string NameWithArticle(this CelestialBody body) {
-    if (!StartsWithCapitalizedDefiniteArticle(body.displayName)) {
-      return body.displayName;
+  public static string Name(this CelestialBody body) {
```

Comment:
Please comment this function, I shouldn't have to read the code to know what it does.